src/commands/mod.rs

```diff
@@ -1443,10 +1443,10 @@ fn cmd_log(ui: &mut Ui, command: &CommandHelper, args: &LogArgs) -> Result<(), C
     {
         ui.request_pager();
         let mut formatter = ui.stdout_formatter();
-        let mut formatter = formatter.as_mut();
+        let formatter = formatter.as_mut();
 
         if !args.no_graph {
-            let mut graph = get_graphlog(command.settings(), &mut formatter);
+            let mut graph = get_graphlog(command.settings(), formatter.raw());
```

Comment:
Alternatively, we can add `ui.raw_stdout() -> Box<dyn Write + '_>`.

Reply:
I think it's nice that the mutable borrow means that you can't use `formatter` while you've borrowed `raw()`. What would the advantage of getting another instance from `ui` be?

Reply:
Could avoid copy-pasting `fn raw()`. I'm okay with either way.

Unrelated: grepping ui.new_formatter (or get_graphlog) found two more call sites.

Reply:
Thanks for catching those. I should have caught them myself.